navit/vehicleprofile.h

```diff
@@ -29,6 +31,39 @@ enum maxspeed_handling {
     maxspeed_ignore = 2,		/*!< Ignore maxspeed of segment, always use {@code route_weight} of road profile */
 };
 
+enum emissionclass {
+    //Truck
+    Euro_0,
+    Euro_1,
+    Euro_2,
+    Euro_3,
+    Euro_4,
+    Euro_5,
+    Euro_6,
+    //Car, use label colors and Euro_6d_TEMP for now
+    Cat_RED,
+    Cat_YLW,
+    Cat_GRN,
+    Euro_6d_TEMP,
+    EMISSION_CLASS_MAX,
+};
+
+static struct emissionclasstxt {
+  char  *classes[EMISSION_CLASS_MAX];
+} ectxt = {
+            "Euro 0",
+            "Euro 1",
+            "Euro 2",
+            "Euro 3",
+            "Euro 4",
+            "Euro 5",
+            "Euro 6",
+            "Red Label",
+            "Yellow Label",
+            "Green Label",
+            "Euro 6d temp"
+};
+
```

Comment:
This seems to be really specific to Germany, what about other country's?

Reply:
The Euro x classes are valid in EU, not only in Germany. For the labels you are right. 
The purpose of the emission class was to use it for routing with the low emission zones. As the access tag for the lezs is missing often, it is not usable at the moment. That's why I added the "Enter low emission zone" value that is used for routing now.